src/mon/NVMeofGwMap.cc

```diff
@@ -430,6 +430,62 @@ void  NVMeofGwMap::find_failover_candidate(
   }
 }
 
+void  NVMeofGwMap::handle_gw_performing_fast_reboot(const NvmeGwId &gw_id,
+     const NvmeGroupKey& group_key, bool &map_modified)
+{
+  bool modified = false;
```

Comment:
Why not just pass map_modified to fsm_handle_gw_fast_reboot?

Reply:
sure